config/routes.rb

```diff
@@ -1,16 +1,20 @@
 Rails.application.routes.draw do
-  scope ':locale', locale: /#{I18n.available_locales.join('|')}/ do
-    # resources :projects, only: [:index, :show]
-    resources :people, path: 'team', only: [:index, :show]
-    resources :contact_forms, path: 'contact-forms', only: [:new, :create]
-    get :cieszyn, to: redirect('cieszyn/welcome')
+  defaults locale: :pl do
+    get :service, controller: 'cieszyn/welcome', action: :show
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.